### PR TITLE
Improve modal styles on mobile

### PR DESCRIPTION
**Pull request by elias-garcia**
### What does this PR does?

This PR improves the modal styles on mobile devices as users were not able to type the "I understand" phrase check on smaller devices due to the big margin top on them.

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Respect code style and lint
- [x] Update documentation (if needed)

### CODE REVIEW

File: src/views/shared/confirmation-modal/confirmation-modal.styles.ts [hunk 1]
```diff
@@ -4,12 +4,15 @@ import { Theme } from "src/styles/theme";
```
Comment (amonsosanz) [inline src/views/shared/confirmation-modal/confirmation-modal.styles.ts:4]: the 480 breakpoint is broken

image

File: src/views/shared/confirmation-modal/confirmation-modal.styles.ts [hunk 2]
```diff
@@ -4,12 +4,15 @@ import { Theme } from "src/styles/theme";
 
 export const useConfirmationModalStyles = createUseStyles((theme: Theme) => ({
```
Comment (amonsosanz) [inline src/views/shared/confirmation-modal/confirmation-modal.styles.ts:5]: Under 330px the button is not properly rendered

image

File: src/views/shared/network-box/network-box.styles.ts [hunk 3]
```diff
@@ -29,10 +29,16 @@ export const useNetworkBoxStyles = createUseStyles((theme: Theme) => ({
     alignItems: "center",
```
Comment (amonsosanz) [inline src/views/shared/network-box/network-box.styles.ts:29]: The buttons of the box should wrap:

image

File: src/views/shared/confirmation-modal/confirmation-modal.styles.ts (hunk 2 above)
Comment (elias-garcia) [reply src/views/shared/confirmation-modal/confirmation-modal.styles.ts:5]: No mobile phone has 330px width nowadays... I took as a reference a Pixel 5 (a mobile phone from 3 years ago) and an iPhone SE (released in 2016) to test it and it works fine on that mobiles. Same for the comment below.

File: src/views/shared/confirmation-modal/confirmation-modal.styles.ts (hunk 1 above)
Comment (elias-garcia) [reply src/views/shared/confirmation-modal/confirmation-modal.styles.ts:4]: Mmmm what do you mean by broken? Which device where you trying to test? It seems that it has very low height.

Comment (amonsosanz) [reply src/views/shared/confirmation-modal/confirmation-modal.styles.ts:4]: I mean that I can not scroll down and see the complete modal. I'm using Chrome Desktop and reducing the width to 480.

File: src/views/shared/confirmation-modal/confirmation-modal.styles.ts (hunk 2 above)
Comment (amonsosanz) [reply src/views/shared/confirmation-modal/confirmation-modal.styles.ts:5]: ok, if we don't care about these resolutions, it's all ok. Approved.

**Review by amonsosanz — APPROVED** [review]

**Pull request merged**